src/components/ClusterNodes/ClusterTerminal.tsx

```diff
@@ -985,7 +976,7 @@ export default function ClusterTerminal({
                 selectionListData={selectionListData}
                 socketConnection={socketConnection}
                 setSocketConnection={setSocketConnection}
-                className={`${fullScreenClassWrapper} ${nodeDetailsPageClassWrapper} ${clusterDetailsPageClassWrapper}`}
+                className={`${fullScreenClassWrapper} ${nodeDetailsPageClassWrapper} ${clusterDetailsPageClassWrapper} ${isNodeDetailsPage ? '' : 'dc__border-top'}`}
```

Comment:
Needs to replace Double quotes in classname everywhere

Reply:
??